src/app/projects/project-details/project-details-build/project-details-build.component.html

```diff
@@ -1,40 +1,47 @@
-<p-table
-  class="build-list"
-  [value]="project?.buildHistory"
-  [lazy]="true"
-  (onLazyLoad)="loadBuildsLazy($event)"
-  [paginator]="true"
-  [rows]="rowsCount"
-  [totalRecords]="buildsTotalCount">
+<p-table class="build-list"
+         [value]="project?.buildHistory"
+         [lazy]="true"
+         (onLazyLoad)="loadBuildsLazy($event)"
+         [paginator]="true"
+         [rows]="rowsCount"
+         [totalRecords]="buildsTotalCount">
   <ng-template pTemplate="header">
     <tr>
       <th class="col-id">ID</th>
       <th class="col-stage">STAGE</th>
       <th class="col-environment">ENVIRONMENT</th>
       <th class="col-status">STATUS</th>
+      <th class="col-tests">TEST RESULTS</th>
       <th class="col-branch">BRANCH</th>
       <th class="col-started">STARTED</th>
       <th class="col-duration">DURATION</th>
     </tr>
   </ng-template>
 
-  <ng-template pTemplate="body" let-build>
-    <tr role="button" (click)="navigateToBuild(build)">
+  <ng-template pTemplate="body"
+               let-build>
+    <tr role="button"
+        (click)="navigateToBuild(build)">
       <td class="col-id">{{build.buildId}}</td>
       <td class="col-stage">{{build.stage?.toUpperCase()==="ALL" ? "Run All" : build.stage}}</td>
       <td class="col-environment">{{build.environment}}</td>
       <td class="col-status">
         <span [class]="this.getStatusCssClass(build.statusId)">{{build.status}}</span>
-        <p-progressSpinner
-          *ngIf="isBuildRunning(build)"
-          class="status-progress-spinner"
-          strokeWidth="4"
-          ></p-progressSpinner>
+        <p-progressSpinner *ngIf="isBuildRunning(build)"
+                           class="status-progress-spinner"
+                           strokeWidth="4">
+        </p-progressSpinner>
+      </td>
+      <td class="col-tests">
+        {{this.numberWithCommas(build.testResultListSummary.failed)}}<i class="pi pi-times"></i>&nbsp;
```

Comment:
Usually `this` isnt use in templates. What happened here?

Reply:
Just me not being used to angular.
The `this` references I've introduced should be gone now I think.